app/controllers/katello/api/v2/upstream_subscriptions_controller.rb

```diff
@@ -0,0 +1,46 @@
+require 'katello/resources/candlepin'
+
+module Katello
+  class Api::V2::UpstreamSubscriptionsController < Api::V2::ApiController
+    before_action :find_organization
+
+    resource_description do
+      description "Red Hat subscriptions management platform."
+      api_version 'v2'
+    end
+
+    def_param_group :cp_pagination do
+      param :page, :number, :desc => N_("Page number, starting at 1")
+      param :per_page, :number, :desc => N_("Number of results per page to return")
+    end
+
+    api :GET, "/organizations/:organization_id/upstream_subscriptions",
+      N_("List available subscriptions from Red Hat Subscription Management")
+    param :organization_id, :number, :desc => N_("Organization ID"), :required => true
+    param_group :cp_pagination
+    def index
```

Comment:
@waldenraines here is the API description. It supports 3 params: `page`, `per_page`, and `organization_id`. You can see that pagination is slightly different than other API endpoints, as it passes the pagination params directly to the candlepin API.

Reply:
we should try to make it as consistent with our api as possible, even if that means translating it in the controller.